packages/swap/src/routes/__tests__/v2/swap.test.ts

```diff
@@ -1239,7 +1246,39 @@ describe('server', () => {
       });
       expect(body.boost).toMatchObject({
         effectiveBoostFeeBps: 5,
-        maxBoostFeeBps: 30,
+        maxBoostFeeBps: 7,
```

Comment:
changed because value in `DepositBoosted` overwrites value from `SwapDepositAddressReady`